generators/app/templates/src/app/components/SearchBar/constants.js

```diff
@@ -0,0 +1 @@
+export const SEARCH_FORM = 'SEARCH_FORM';
```

Comment:
remove this, so we allow the user to define any number of form names instead of only this one, and for the same reasons as I stated in the other comment about reduxForm